okhttp/src/main/java/okhttp3/internal/platform/Platform.java

```diff
@@ -75,10 +76,10 @@
  * <p>Supported on Android 6.0+ via {@code NetworkSecurityPolicy}.
  */
 public class Platform {
+  private static final Logger logger = Logger.getLogger(OkHttpClient.class.getName());
```

Comment:
Please revert this move.

Reply:
If the logger is initialized after `private static final Platform PLATFORM`, then `this.logger` will be null in `Platform.isConscryptPreferred()`.
So an NullPointerException will be thrown on line 196 when `Platform.isConscryptPreferred()` calls `logger.log(...)`.

Would you prefer to use another instance of the logger, or to not log this exception at all?

Reply:
To unblock this and given our policy on public APIs how about using a literal string here and commenting why.

Reply:
Your explanation seems reasonable.  I suggest leaving it as you now have it.

Reply:
By moving the try-catch block into `okhttp.internal.Util.getSystemProperty`, it's no longer needed to have the logger initialized in `Platform.isConscryptPreferred()`, so I reverted this change.